.gitignore

```diff
@@ -1,10 +1,3 @@
+/.idea/
 inc/config.inc.php
-.DS_Store
```

Comment:
There were so many entries for me without function. Would be good to comment them with their sense. Do we only need the .DS_Store back or all?